modules/ossm-vs-istio.adoc

```diff
@@ -85,12 +85,16 @@ spec:
 
 {SMProductName} replaces BoringSSL with OpenSSL. OpenSSL is a software library that contains an open source implementation of the Secure Sockets Layer (SSL) and Transport Layer Security (TLS) protocols. The {SMProductName} Proxy binary dynamically links the OpenSSL libraries (libssl and libcrypto) from the underlying Red Hat Enterprise Linux operating system.
 
-
 [id="ossm-external-workloads_{context}"]
 == External workloads
 
 {SMProductName} does not support external workloads (virtual machines).
 
+[id="ossm-virtual-machine-workloads_{context}"]
+== Virtual Machine Workloads
+
+You can deploy virtual machines into OpenShift, and then integrate them into a mesh using OpenShift Virtualization.
```

Comment:
This is confusing to me. The section above says we do not support it. Now we say users can deploy virtual machines. Are both statements true? Are they referring to the same or different concepts?

Reply:
@jwendell These are referring to different concepts. 

Here is my take: My basic understanding is that this entry is trying to communicate that from OpenShift you can integrate VM workloads into a mesh using OpenShift Virtualization, and the prior comment is trying to communicate that you cannot have the mesh connect directly to the VM workloads.  So these are two different use cases. (Again, this my understanding from a conversation; I haven't seen this work.)

I'd like to hear your thoughts. Hopefully this is enough information to help convey the intent of what we're trying to say. If not, I can chat with Rob after he is back or possibly Jamie to see how we might be able to clarify things. I'm also open to any suggestions you might have.